server/db/migration/V20220413.2005__adjust_tooltips_1059.sql

```diff
@@ -0,0 +1,112 @@
+ that bicycle, walk, and take transit. All PROW investment shall be consistent 
+ with the Mobility Plan 2035, and may include, but are not limited to, curb extensions, 
+ leading pedestrian intervals, controlled mid-block crosswalks, pedestrian refuge 
+ islands, protected bicycle lanes, bike boxes, exclusive bicycle signal phases, 
+ street trees, etc. LADOT shall be consulted to verify the opportunity and feasibility 
+ of access improvements near the project site.. Point value relative to improvement 
+ and location, and determined in coordination with LADOT staff.'
+where calculationId = 1 and code = 'STRATEGY_MOBILITY_INVESTMENT_1'
+
+update CalculationRule set 
+ description = 'Implement a “cash out” program, where all full or 
+ part-time employees who do not use a parking space are paid the 
+ value of the space instead in time increments that the parking is leased. 
+ The value of a space shall be the leased value, if leased, and shall be the 
+ market value of a parking space if owned by the property owner.'
+where calculationId = 1 and code = 'STRATEGY_PARKING_2'
+
+update CalculationRule set 
+ description = 'Provide public access to the property''s parking. 
+ Must be coupled with on-demand parking availability publicized 
+ through public signage and/or approved mobile application. This 
+ strategy is especially encouraged for properties that provide parking 
+ supply at rates above L.A.M.C. or a Specific Plan requirements. To earn 
+ points for this strategy, a project must provide the number of parking 
+ spaces available for public use. That supply must be, at a minimum, 
+ 25% of the total parking supply rounded up to the next whole number.'
+where calculationId = 1 and code = 'STRATEGY_PARKING_4'
+
+update CalculationRule set 
+ description = 'Partner with a shared micro-mobility company to provide 
+ discounted membership fees for building occupants 
+ (e.g. residents and employees) Make shared micro-mobility 
+ fleet devices accessible for easy identification and use.'
+where calculationId = 1 and code = 'STRATEGY_SHARED_MOBILITY_1'
+
+update CalculationRule set 
+ description = 'Purchase and operate a shared micro-mobility fleet that 
+ is available on-site for use or rent for building occupants 
+ (e.g. residents and employees). The fleet size shall be determined 
+ to ensure a shared device is available 90 percent of the 
+ time it is requested.'
+where calculationId = 1 and code = 'STRATEGY_SHARED_MOBILITY_2'
+
+update CalculationRule set 
+ description = 'Provide funding to a local transit provider for 
+ improvements that improve service quality (reduce headways, etc.) 
+ at transit stops within ¼ mile radius of the project site. Funds 
+ could also contribute to an existing shuttle or microtransit 
+ service (e.g. DASH) in consultation with LADOT if this option 
+ is available near the project site.'
+where calculationId = 1 and code = 'STRATEGY_TRANSIT_ACCESS_4'
```

Comment:
These lines are extras from V20210526.1439__adjust_tooltips_827_832.sql. They can be safely deleted.
```suggestion
```